docker/pytorch/Dockerfile

```diff
@@ -3,12 +3,16 @@ ARG BASE_IMAGE
 FROM ${BASE_IMAGE}
 ARG DEBIAN_FRONTEND=noninteractive
 
+# remove a bad symlink
+RUN rm -f /usr/local/cuda-11.3/cuda-11.3  
```

Comment:
What's going on here?  Is this something vestigial from the Nvidia CUDA container or is it conflicting with the NCCL workaround below?

Reply:
It's a bad file from the nvidia image